translations/da/get5.phrases.txt

```diff
@@ -6,6 +6,7 @@
     }
     "WaitingForCastersReadyInfoMessage"
     {
+        "#format"       "{1:s}"
```

Comment:
Why are these being added? They should typically only be on the english file: https://wiki.alliedmods.net/Translations_(SourceMod_Scripting)#Distributing_Language_Files

Reply:
Didn't know. I'll revert that.